local-setup/docs/TENANT-ONBOARDING.md

```
@@ -0,0 +1,256 @@
+# local-setup/ansible/inventory/host_vars/<tenant>.yml
+nginx_features:
+  configurator: true         # renders the /configurator/ nginx location
+build_configurator: true     # clone + `vite build` the SPA at deploy time
+```
+
+Re-run `./deploy.sh <tenant>` after flipping these. See
+[`../ansible/README.md`](../ansible/README.md) for the full deploy flow.
+
+### Open it
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)